CONTRIBUTING.md

```diff
@@ -59,25 +68,55 @@ Code review is a crucial aspect of contributing to a project, and all contributo
 
 ## Design document
 
-Write down your implementation plan and discuss it with other developers in the community before you start coding officially.
-If it is just a small changes, describe the implementation steps clearly in the Issue.
-If it is a relatively large work, it is recommended to write a design document for this feature.
-Here is a [design document template](https://docs.google.com/document/d/1LeTyrlzQJfSs2DkRBsucK_vV5gtHRYLb1KSrpu0hp3g/edit?usp=sharing) for reference.
+Write down your implementation plan and discuss it with other developers in the community before you
+start coding officially. If it is just a small change, describe the implementation steps clearly in
+the Issue. If it is a relatively large work, it is recommended to write a design document for this
+feature. Here is
+a [design document template](https://docs.google.com/document/d/1LeTyrlzQJfSs2DkRBsucK_vV5gtHRYLb1KSrpu0hp3g/edit?usp=sharing)
+for reference.
 
 ## Code suggestions
 
-### Code style
-If you are using IntelliJ IDEA, you can import code style configurations by following these steps:
-1. Open Settings → Editor → Code Style → Show Schema Actions → Import Schema → IntelliJ IDEA Code Style XML
-2. Select the `{ARCTIC_PROJECT_HOME}/style/idea/intellij-java-palantir-style.xml` file
-3. Click OK to confirm.
+### Code formatting
+
+Amoro uses [Spotless](https://github.com/diffplug/spotless/tree/main/plugin-maven) together with
+[google-java-format](https://github.com/google/google-java-format) to format the Java code. For
+Scala, it uses Spotless with [scalafmt](https://scalameta.org/scalafmt/).
+
+You can format your code by executing the command `mvn spotless:apply` in the root directory of
+project.
+
+Or you can configure your IDEA to automatically format your code. Then you will need to install
+the [google-java-format](https://github.com/google/google-java-format) plugin. However, a specific
+version of this plugin is required.
+Download [google-java-format v1.7.0.6](https://plugins.jetbrains.com/plugin/8527-google-java-format/versions/stable/115957)
+and install it as follows. Make sure to never update this plugin.
+
+1. Go to “Settings/Preferences” → “Plugins”.
+2. Click the gear icon and select “Install Plugin from Disk”.
+3. Navigate to the downloaded ZIP file and select it.
+
+After installing the plugin, format your code automatically by applying the following settings:
+
+1. Go to “Settings/Preferences” → “Other Settings” → “google-java-format Settings”.
+2. Tick the checkbox to enable the plugin.
+3. Change the code style to “Default Google Java style”.
+4. Go to “Settings/Preferences” → Editor → Code Style → Scala.
+5. Change the “Formatter” to “scalafmt”.
+6. Go to “Settings/Preferences” → “Tools” → “Actions on Save”.
+7. Under “Formatting Actions”, select “Optimize imports” and “Reformat file”.
```

Comment:
Should `Reformat file` be `Reformat code` ? (My IntelliJ IDEA 2021.2.3 (Community Edition))
<img width="1819" alt="image" src="https://github.com/NetEase/amoro/assets/103108928/00ae6388-52b8-4697-aac9-2e7de424254e">